### PR TITLE
Fix panic with nil github issue number in config

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: cmd/publishing-bot/config/config.go [hunk 1]
```diff
@@ -32,5 +32,5 @@ type Config struct {
 	DryRun bool
 
 	// A github issue number to report errors
-	GithubIssue *int `yaml:"github-issue,omitempty"`
+	GithubIssue int `yaml:"github-issue,omitempty"`
```
Comment (mfojtik) [inline cmd/publishing-bot/config/config.go]: maybe make this a string and if empty don't post anything? allows more notification delivery tools

Comment (sttts) [reply cmd/publishing-bot/config/config.go]: We can change that later when we add another delivery method. Would keep it simple here, just to fix the panic.

Comment (mfojtik) [reply cmd/publishing-bot/config/config.go]: fine with that

**Pull request merged**